drivers/hmis/app/models/hmis/ce/match/expression/client_field_map.rb

```diff
@@ -118,6 +118,6 @@ def project_types_query(clients, scope, project_association)
     # display helpers
     def helpers = ApplicationController.helpers
     def format_days(days) = days.nil? ? nil : helpers.pluralize(days, 'day')
-    def format_project_types(project_type_ids) = project_type_ids.uniq.map { |t| HudUtility2026.project_type(t) }
+    def format_project_types(project_type_ids) = project_type_ids.uniq.map { |t| HudHelper.util('2026').project_type(t) }
```

Comment:
```suggestion
    def format_project_types(project_type_ids) = project_type_ids.uniq.map { |t| HudHelper.util.project_type(t) }
```
Unless I'm missing some reason to specify 2026 here